content/en/docs/how-tos/interact-with-running-jobs.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Interacting With CI Job Currently Running and Tests Binary"
+date: 2020-12-15T16:08:01-04:00
+draft: false
+description: How to interact with the CI cluster to follow a job currently running your Work In Progress (WIP) Pull Request (PR).
+---
+
+## Overview
+
+The OpenShift CI system runs on OpenShift clusters and many jobs that run in OpenShift repository PRs install
+external clusters installed with a release payload that is built with any changes you made. Test suites are then
+run against the external clusters from a running pod within the CI cluster. It is possible to follow the logs of
```

Comment:
It could be useful to give the two types of clusters some names ("CI cluster", "tested cluster" or something similar) and use them consistently in the docs? The text mentions a "cluster" a lot and I think giving them some defined names would make reading a bit easier.

Reply:
good suggestion, I'll clean that up/clarify

Reply:
done
"CI cluster", "test cluster" and "developement cluster" with Note in Overview